src/components/layout/dashboard/DashboardContentContainer.tsx

```diff
@@ -45,7 +46,7 @@ const DashboardContentContainer: React.FC<Props> = ({
   contentSx,
 }) => {
   const theme = useTheme();
-  const maxPageWidth = theme.breakpoints.values.xl;
+  const maxPageWidth = useMaxPageWidth();
```

Comment:
`FormBuilder.tsx` was already defining its own `maxPageWidth` that was supposed to match the styles used by `DashboardContentContainer.tsx` (see the [comment](https://github.com/greenriver/hmis-frontend/pull/903/files#diff-d278cce8d05aa45f7d08250bb57d807cc320462c5e0ef165263b095bf4561a7dR140-R141)). But for some reason it was set to `lg` instead of `xl`. I think they should be kept consistent, so I factored it out into a common hook.